common/arg.cpp

```diff
@@ -3459,8 +3467,8 @@ common_params_context common_params_parser_init(common_params & params, llama_ex
             params.model.hf_repo = "ggml-org/Qwen2.5-Coder-1.5B-Q8_0-GGUF";
             params.model.hf_file = "qwen2.5-coder-1.5b-q8_0.gguf";
             params.port = 8012;
-            params.n_gpu_layers = 99;
-            params.flash_attn = true;
+            params.n_gpu_layers = 999;
+            params.flash_attn_type = LLAMA_FLASH_ATTN_TYPE_ENABLED;
```

Comment:
Since these values are the same as the default now, these lines could be removed entirely.